programs/kmemleak-test/parse

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
```

Comment:
thanks, for this file, need link to original stats/kmemleak instead of making of copy

Reply:
ok